shell/edit/provisioning.cattle.io.cluster/index.vue

```diff
@@ -266,8 +271,7 @@ export default {
     subTypes() {
       const getters = this.$store.getters;
       const isImport = this.isImport;
-      const isElementalActive = !!this.activeProducts.find((item) => item.name === ELEMENTAL_PRODUCT_NAME);
-
+      const isElementalActive = !!getters['type-map/activeProducts'].find((item) => item.name === ELEMENTAL_PRODUCT_NAME);
```

Comment:
@aalves08 Not sure why this change - any ideas? I think this should probably be removed - doesn't seem relevant to the overall change.

Reply:
This doesn't make sense. I ran this branch, but with the original `const isElementalActive = !!this.activeProducts.find((item) => item.name === ELEMENTAL_PRODUCT_NAME);` and all was fine with cluster creation and Elemental.

I've reverted the change and pushed the code @nwmac

Reply:
Ok, this change is because of the unit test created... 💦 it's because Richa must have hit a roadblock when mocking a namespaced getter `mapGetters('type-map', ['activeProducts'])`.

Let me see if there's an alternative.

Reply:
Unit test fixed. Pushed changes